.github/workflows/security.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: security
+
+on:
+  push:
+    branches: [development/*, stabilization/*, hotfix/*]
+  release:
+    types: [published]
+  pull_request:
+    branches: [development/*, stabilization/*, hotfix/*]
+  schedule:
+    - cron: '0 8 * * 1' # Monday - 8am - UTC
+  workflow_dispatch:
+
+
+jobs:
+  build-zenko-ui:
+    permissions:
+      contents: read
+      packages: write
+    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
+    with:
+      name: zenko-ui
```

Comment:
Instead of rebuilding the image, we can run the scan on the image matching the head of the branch.

Reply:
We might be facing a race condition in this case. If we want this workflow to be independent we should be building this image.

Reply:
You mean using the build that is created in `build-push-tests.yml`, right? In this case, indeed, we may have a race condition issue since we cannot use cross-file `need`.